CMakeLists.txt

```diff
@@ -1163,26 +1164,38 @@ function(print_target_properties target)
 endfunction()
 
 if(ENABLE_HOST_CPU_DEVICES AND NOT ENABLE_CONFORMANCE)
+
   find_package(onnxruntime QUIET)
   set(HAVE_ONNXRT ${onnxruntime_FOUND})
   if(${HAVE_ONNXRT})
-    print_target_properties(onnxruntime::onnxruntime)
     # TODO: this is a hack, just passing onnxruntime::onnxruntime
     # to include_directories should probably work
     get_target_property(onnxruntime_INCLUDE_DIRS onnxruntime::onnxruntime INTERFACE_INCLUDE_DIRECTORIES)
     get_target_property(onnxruntime_IMPORTED_CONFIGURATIONS onnxruntime::onnxruntime IMPORTED_CONFIGURATIONS)
-    message("Found ONNXRuntime headers at ${onnxruntime_INCLUDE_DIRS} for configs ${onnxruntime_IMPORTED_CONFIGURATIONS}")
+    message(STATUS "ONNXRuntime version ${onnxruntime_VERSION} found, with headers at ${onnxruntime_INCLUDE_DIRS} for configs ${onnxruntime_IMPORTED_CONFIGURATIONS}")
   else()
     message(STATUS "ONNXRuntime not found, disabled")
   endif()
 
-  pkg_search_module(LIBJPEG_TURBO IMPORTED_TARGET
-          libjpeg-turbo>=3.0.0 libturbojpeg>=3.0.0)
-  if(NOT LIBJPEG_TURBO_FOUND)
+  find_package(LIBJPEG_TURBO 3.0.0 QUIET CONFIG NAMES libjpeg-turbo)
```

Comment:
Should the alternate name (libturbojpeg) still be considered?

Reply:
My source build (for 3.x, since ubuntu 24.04 only packages version 2.1.5) calls the cmake config `libjpeg-turbo` and provides two pkgconfig files `libjpeg.pc` and `libturbojpeg.pc`. This seems to match the ubuntu package as well - minus `libjpeg.pc` which is not provided there. As far as I can tell, `find_package` only looks for the cmake configuration so I don't think there is anything to gain from the alternate name.